repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/shareThisPage.ftl

```diff
@@ -1,18 +1,25 @@
 <#ftl output_format="HTML">
-<#macro shareThisPage document mediaTitle="" mediaLink="" iconPath="">
+<#macro shareThisPage document mediaTitle="" mediaLink="" iconPath="" smOptions={}>
```

Comment:
I do not believe it is, but without amending all the calls to it, which was outside the scope of the Jira, I thought best to not mess with it in case it was used by something.

Reply:
Looks like this is only called from shareSection.ftl, feature.ftl and blog-article.ftl, do you want me to add this change to the Jira?

Reply:
Yes please.

Reply:
Updated:-
shareThisPage.ftl
shareSection.ftl
feature.ftl
blog-article.ftl
removing the unused document argument

Reply:
Thank you